releng/gradle-composite/build.gradle

```diff
@@ -8,15 +9,15 @@ task('generatePrefs') {
             if (!(new File(file, '.settings')).exists()) {
                 new File(file, '.settings').mkdir();
             }
-            (new File(file, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir=../xtext-umbrella/releng/gradle-composite\nproject.path=\\:"+file.name+"\n"
+            (new File(file, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir="+base+"\nproject.path=\\:"+file.name+"\n"
                     
             file.listFiles().each { File fx -> 
                 if (fx.isDirectory() && new File(fx, "build.gradle").exists()) {
                     println  "generating buildship prefs for " + fx.path
                     if (!(new File(fx, '.settings')).exists()) {
                         new File(fx, '.settings').mkdir();
                     }
-                    (new File(fx, '.settings/org.eclipse.buildship.core.prefs')).text = "connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\nconnection.project.dir=../../xtext-umbrella/releng/gradle-composite\nproject.path=\\:"+fx.name+"\n"
```

Comment:
The default setting for Oomph seems to name the repository `xtext-umbrella-master`